src/tagmanager/tm_parser.c

```diff
@@ -1223,14 +1226,11 @@ const gchar *tm_parser_get_sidebar_info(TMParserType lang, gint group, guint *ic
 			return NULL;
 
 		grp = &map->groups[group - 1];
-		name = grp->name;
+		name = _(grp->name);
 		*icon = grp->icon;
 	}
-#ifdef GETTEXT_PACKAGE
-	return g_dgettext(GETTEXT_PACKAGE, name);
-#else
+
 	return name;
-#endif
```

Comment:
The `if (group == 0)` case didn't change because now `_()` means `gettext()`, so it does the translation directly -- hence why not doing it here anymore.